MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include LICENSE
+include README.rst
+
+recursive-include docs *
```

Comment:
docs directory is only created in another issue. You need to add recurvice include for adminapi in, as well as AUTHORS.rst.